projects/common/src/utilities/formatters/date/date-formatter.test.ts

```diff
@@ -28,6 +28,14 @@ describe('Date formatter', () => {
       new DateFormatter({
         mode: DateFormatMode.DateWithYearAndTimeWithTimeZone
       }).format(dateString)
-    ).toEqual('19 Aug 2021 11:35 PM GMT+00:00');
+    ).toEqual('19 Aug 2021 1:02 PM GMT+00:00');
+  });
+
+  test('can format a date string with and time with offset time zone', () => {
+    expect(
+      new DateFormatter({
+        mode: DateFormatMode.TimeWithTimeZoneOffset
+      }).format(dateString)
+    ).toEqual('01:02:03+00:00');
```

Comment:
This expectation is wrong, it's showing the bug I described in my other comment. This time should match the original time if they're both utc, but a better test would be to use a different time zone for input.

Reply:
The tests run in GMT, so the timezone stuff is a little wonky in the tests. Let me update the impl to allow specifying timezones so we can have better tests.